cpp/mcap/include/mcap/mcap.hpp

```diff
@@ -1,12 +1,22 @@
 #pragma once
 
+#include "errors.hpp"
+#include <climits>
```

Comment:
probably prefer `numeric_limits`?

Reply:
`numeric_limits` does not define `CHAR_BIT` AFAIK

Reply:
`numeric_limits<unsigned char>::digits`? https://en.cppreference.com/w/cpp/types/numeric_limits/digits